### PR TITLE
CLOUDP-176392: rename mongodbatlasv2 to admin. Organise imports across repository

### PR DESCRIPTION
**Pull request by wtrocki**
## Description

NOTE: build will fail due to renaming not being pushed to this PR. Rename will bring +800 files changed to the PR and it will rename:

mongodbatlasv2 => admin 

I will push rename after the initial review. All builds are passing locally. Tested makefile changes end to end.

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: tools/config/go-templates/atlas_client.mustache [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{! X-XGEN-CUSTOM }}package {{packageName}} // import "go.mongodb.org/atlas/mongodbatlas/v2"
```
Comment (gssbzn) [inline tools/config/go-templates/atlas_client.mustache:1]: would this comeback with the new url?

Comment (wtrocki) [reply tools/config/go-templates/atlas_client.mustache:1]: I do not understand this syntax (it was inspired by v1 api) thus removed it. 
Can restore that now.

Comment (gssbzn) [reply tools/config/go-templates/atlas_client.mustache:1]: when using vanity urls both `import "go.mongodb.org/atlas-sdk/admin"` and `import "https://github.com/mongodb/atlas-sdk-gok/admin" are valid, this comment `// import "go.mongodb.org/atlas/mongodbatlas/v2"` throws a nice error when trying to use anything but the vanity url

Comment (wtrocki) [reply tools/config/go-templates/atlas_client.mustache:1]: Nice catch. Restoring that now.

**Comment by wtrocki** [comment]
I have noticed that every PR now has +500 files changed due to version. 
Limited version exposure in that PR.
Example: https://github.com/mongodb/atlas-sdk-go/pull/4#discussion_r1189931207

**Pull request merged**